lib/internal/bootstrap/node.js

```diff
@@ -269,25 +269,40 @@
         // If -i or --interactive were passed, or stdin is a TTY.
         if (process._forceRepl || NativeModule.require('tty').isatty(0)) {
           // REPL
-          const cliRepl = NativeModule.require('internal/repl');
-          cliRepl.createInternalRepl(process.env, function(err, repl) {
-            if (err) {
-              throw err;
-            }
-            repl.on('exit', function() {
-              if (repl._flushing) {
-                repl.pause();
-                return repl.once('flushHistory', function() {
-                  process.exit();
-                });
+
```

Comment:
It would be nice to put the block into a function (e.g. `setupRepl` or `setupEval`) to avoid further spaghetti-fy the code here.